GPU/Software/Rasterizer.cpp

```diff
@@ -1451,26 +1451,35 @@ void ClearRectangle(const VertexData &v0, const VertexData &v1)
 	minY = std::max(minY, (int)TransformUnit::DrawingToScreen(scissorTL).y);
 	maxY = std::max(0, std::min(maxY, (int)TransformUnit::DrawingToScreen(scissorBR).y + 16));
 
+	constexpr int MIN_LINES_PER_THREAD = 32 * 16;
```

Comment:
Somewhere around here, we should add a very brief comment explaining the multiplication by 16 to get subpixel precision.

Reply:
Okay, added a comment.

-[Unknown]